Podfile.lock

```diff
@@ -2,11 +2,12 @@ PODS:
   - Alamofire (4.9.1)
   - Alamofire-Synchronous (4.0.0):
     - Alamofire (~> 4.0)
-  - AlphaWalletAddress (1.0.1):
+  - AlphaWalletAddress (1.0.2):
+    - AlphaWalletCore
     - EthereumAddress
     - TrustKeystore
     - web3swift
-  - AlphaWalletCore (1.0.1):
+  - AlphaWalletCore (1.0.2):
```

Comment:
I *think* we can cheat a bit and not worry about version numbers for our inhouse pods until we need to publish them for others. It'll still work right, because the hash is included in `Podfile.lock`? Or does it not work?